<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>631</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ServerLayout">
     <item>
      <widget class="QLabel" name="ServerAddressLabel">
       <property name="text">
        <string>Server Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ServerAddressLineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ServerPortLabel">
       <property name="text">
        <string>Server Port: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ServerPortLineEdit">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>631</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="DestinationLayout">
     <item>
      <widget class="QLabel" name="DestinatioAddressLabel">
       <property name="text">
        <string>Destination Address: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DestinatioAddressLineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DestinationPortLabel">
       <property name="text">
        <string>Destination Port: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DestinationPortLineEdit">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>631</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="FolderLayout">
     <item>
      <widget class="QLabel" name="OriginFolderLabel">
       <property name="text">
        <string>Origin Folder: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OriginFolderLineEdit">
       <property name="text">
        <string>/home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OriginFindPushButton">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DestinationFolderLabel">
       <property name="text">
        <string>Destination Folder: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DestinationFolderLineEdit">
       <property name="text">
        <string>/home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DestinationFindPushButton">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>91</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="StartButtonLayout">
     <item>
      <widget class="QPushButton" name="SendPushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="TittleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>671</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DISTRIBUTED BACKUP &gt;&gt; CLIENT PROGRAM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
